demo-simple/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+  <base-config cleartextTrafficPermitted="true" />
```

Comment:
Starting with Android 9.0 (API level 28), cleartext support is disabled by default. You need to specify this file or do a hack on the application in the manifest to workaround it
The idea being that the app should specify what endpoints it can interact with rather than just connecting to anything :joy:
I specify `true` so that it behaves as it did prior to API 28 and have added it to the demo to show how people can override it